components/patterns/elements/image-loader.tsx

```diff
@@ -0,0 +1,13 @@
+const ImageLoader = ({ src, width }) => {
+  const imageStyles = [
+    {w: 128, s: 'breakpoint_sm_2x'},
+    {w: 384, s: 'breakpoint_md_2x'},
+    {w: 750, s: 'breakpoint_lg_2x'},
+    {w: 1200, s: 'breakpoint_xl_2x'},
+    {w: 3840, s: 'breakpoint_2xl_2x'},
+  ];
+  const style = imageStyles.find(s => s.w === width)?.s || 'breakpoint_xl_2x';
```

Comment:
Writing this out in English so my brain is happy.

For all images, that are not local or relative, try and match an image style to an image width, if no matching image width to the array of imageStyles, default to the biggest image style.

Reply:
Example: https://github.com/SU-SWS/sulgryphon-nextjs/blob/1.x/components/paragraph/stanford-card.tsx#L43

That wouldn't get all the src set values (128, 384, 750, 1200, 3840) because it doesn't set the `sizes` key? It would just render as the one size?

Reply:
Gotcha thanks.

So the convention and recommendation would be to supply a sizes property wherever images are being used, otherwise the largest image will be used?

Reply:
no no... It's recommended to use the default values. Maybe unless you are displaying a small image and don't need the large versions. If you don't pass a `sizes` property when using an `<Image>` it will make a responsive image using the sizes defined in the next.config.js which should be the most ideal in most situations I think.

Reply:
Gotcha, ok, this info is something that @jenbreese will find helpful/useful when she gets going on the SUP project.